README.md

```diff
@@ -2,3 +2,12 @@
 
 This repository contains code for BxJS Website.  
 Primary website content is [BxJS Weekly](https://github.com/BuildingXwithJS/bxjs-weekly) podcast.
+
+## Install with Yarn
+* `npm install -g yarn`
```

Comment:
This is not a recommended way to install yarn: https://classic.yarnpkg.com/en/docs/install#windows-stable

Reply:
well, that's was the fatest way to install it.

Reply:
Not the recommended way though, so if you think that's really needed - I'd just link the official guide

Reply:
I put it because I stuck my head against wall trying to install the project with npm, so it just to provide the fast path to get the project set up.

Reply:
I get that, but I don't think it's a good idea to provide non-recommended way in readme. Especially considering the nature of BxJS content